### ISSUE TITLE
Windows Phone scanner makes continues shutter sound

### ISSUE THREAD

**Issue by SunboX** [phonegap/phonegap-plugin-barcodescanner#119]
The Windows Phone (`cordova add platform windows`) scanner makes a continues shutter sound until it recognizes a bar code. There are other Windows Phone apps that do not seem to need this 'technique' of continuous picture taking. Someone with experience in WinJS/C# should take a look at this.

Maybe this one is helpful to fix this issue:
https://github.com/mmaitre314/VideoEffect#realtime-video-analysis-and-qr-code-detection

**Comment by SunboX** [comment]
related to https://github.com/phonegap/phonegap-plugin-barcodescanner/issues/85

**Comment by lj222kj** [comment]
Have you tried cordova add platform wp8 ?

**Comment by lj222kj** [comment]
Hi

Have you found a solution to your problem?

**Comment by SunboX** [comment]
No, there's no simple solution. Someone has to extend the windows universal cordova scanner and use the Nokia Imaging SDK to get a still image from the camera video stream.

**Comment by SunboX** [comment]
You can use cordova wp8 platform, but that is not universal.

**Comment by lj222kj** [comment]
By universal, do you mean all windows phone devices or?

**Comment by SunboX** [comment]
:arrow_right: https://msdn.microsoft.com/en-US/library/windows/apps/dn726767.aspx

**Comment by SunboX** [comment]
There's someone from Xamarin working on the ZXing.NET Windows Universal sample. This may help implementing the Cordova plugin: https://github.com/Redth/ZXing.Net.Mobile/tree/master/Samples/WindowsUniversal

**Comment by vladimir-kotikov** [comment]
@SunboX, seems they don't support [Windows 8/8.1/Phone 8.1](https://github.com/Redth/ZXing.Net.Mobile/blob/master/Source/ZXing.Net.Mobile.WindowsRT/NotSupported.cs#L12).

As for universal apps, the Barcode Reader now uses `getPreviewFrameAsync` to get video frames without any delay and shutter sound - see https://github.com/phonegap/phonegap-plugin-barcodescanner/commit/521d427ed418d2288a1b31a11398b99f54fb1f5d
